UnitystationLauncher/UnitystationLauncher.csproj

```diff
@@ -1,7 +1,7 @@
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <OutputType>WinExe</OutputType>
-    <TargetFramework>net7.0</TargetFramework>
+    <TargetFramework>net8.0-windows</TargetFramework>
```

Comment:
The target framework should be `net8.0`, not `net8.0-windows`, as this application is cross platform between Windows, MacOS, and Linux.